javaslang/src/test/java/javaslang/collection/AbstractSeqTest.java

```diff
@@ -1064,6 +1065,56 @@ public void shouldNotRemoveAllNonExistingElementsFromNonNil() {
         }
     }
 
+    // -- removeAll(Predicate)
+
+    @Test
+    public void shouldRemoveExistingElements() {
+        assertThat(of(1, 2, 3).removeAll(i -> i == 1)).isEqualTo(of(2, 3));
+        assertThat(of(1, 2, 3).removeAll(i -> i == 2)).isEqualTo(of(1, 3));
+        assertThat(of(1, 2, 3).removeAll(i -> i == 3)).isEqualTo(of(1, 2));
+        if (useIsEqualToInsteadOfIsSameAs()) {
+            assertThat(of(1, 2, 3).removeAll(ignore -> true)).isEmpty();
```

Comment:
this can be removed because of `shouldNotRemoveAllNonMatchedElementsFromNonNil()`